packages/calendar-input/src/types.ts

```diff
@@ -0,0 +1,53 @@
+import {TextInputAppearanceType, TextInputProps} from '@smashing/text-input'
+export type CalendarInputAppearanceType = 'default' | 'outline'
+
+export type StyledCalendarInputProps = {
+  appearance: CalendarInputAppearanceType
+}
+export type CalendarInputProps = {
+  appearance: CalendarInputAppearanceType
+  inputAppearance?: TextInputAppearanceType
+  onChange?: (date?: Date) => void
+  value?: Date
+  withTime?: boolean
```

Comment:
```suggestion
  hasTime?: boolean
```
In most components `has` prefix is used, let's keep it that way